pkg/tenant/db/members.go

```diff
@@ -26,7 +26,7 @@ const (
 
 var (
 	// Must be at least 3 characters, cannot start with a number, and is alphanumeric with + _ and -
-	WorkspaceNameRegex = regexp.MustCompile(`^[a-zA-Z][a-zA-Z0-9+_-]{2,}$`)
+	WorkspaceNameRegex = regexp.MustCompile(`^[a-zA-Z][a-zA-Z0-9.+_-]{2,}$`)
```

Comment:
what prompted this? Are `.` allowed in slugs?

Reply:
I'm not sure we can allow `.` if we're going to use this for subdomains in the future.

Reply:
My bad, somehow I thought I'd just overlooked this in the last PR.